docs/reference/tags/create_line_item.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Create Line Item
+parent: Tags
+has_children: false
+---
+
+The Create Line Item Tag renders a form which acts as a wrapper for a single item.
+
+##### input
+{% raw %}
+```liquid
+    {% form "create_line_item" %}
+        <input type="hidden" data-variant-id="{{item_id}}" name="items[][variant_id]" value="{{item_id}}"/>
+    {% endform %}
```

Comment:
There are a few more fields for the create line item form, could those also be included?

```
create_line_item

{% form 'create_line_item' %}
  <input type="hidden" name="items[][quantity]" value="**<quantity>**" />
  <input type="hidden" name="items[][variant_id]" value="**<variant or extra id>**" />

	// per unit variants also expect an adult count
  <input type="hidden" name="items[][adult_count]" value="**<adult count>**" />

	// where modifiers are included these are seperate lines
  <input type="hidden" name="items[][modifier_ids][]" value="**<modifier id>**" />
  <input type="hidden" name="items[][modifier_ids][]" value="**<modifier id>**" />

  <input type="submit" value="Add to cart" />
{% endform %}
```

Reply:
Thanks for your example!

This form was tricky since we don't have spec tests, so I've had to make some assumptions about how it works. 

Can I double-check...
1. is the first line, which shows quantity in both the `name` and `item` fields is correct
2. Do the two modifier lines represent two separate modifiers for one item, and can my example just include one line?

Reply:
1. Yes I think so - there is an example [here](https://github.com/easolhq/easol/blob/6121973f37bec01d0f77e8c76cffb06b57dde5f9/spec/features/site_cart_interaction_spec.rb)
2. Yes for separate modifiers we typically put those on separate lines. The example could definitely just include one, but perhaps with a note that additional modifiers should be added separately, so developers know how to handle those instances.